JavaScript/Topics/5.Syntax/2.Operators/2.Arithmetic_Operators/README.md

```diff
@@ -0,0 +1,117 @@
+# Arithmetic Operators
+[Back to Operators in Javascript](../README.md#operators-in-javascript)
+
+
+#### 1. Addition (+)
+The addition operator (+) is used to add two given variables.
+```js
+console.log(5+8);
+// expected output: 13
+
+console.log(5+8+3);
+// expected output: 16
+```
+
+#### 2. Subtraction (-)
+The subtraction operator (-) is used to subtract one variable from another.
+```js
+console.log(8-5);
+// expected output: 3
+
+console.log(11-4);
+// expected output: 7
+```
+
+#### 3. Multiplication (*)
+The multiplication operator (*) is used to multiply two variables.
+```js
+console.log(5*8);
+// expected output: 40
+```
+
+#### 4. Division (/)
+The division operator (/) is used to divide one variable by another.
+```js
+console.log(8/3);
+// expected output: 2
```

Comment:
```suggestion
// expected output: 2.6666666666666665
```
I am pointing calculation to this variable only, correct in others if they are incorrect

Reply:
Sir, I am getting 2 here. I have also checked rest all the other calculations.
![Screenshot from 2021-09-22 09-40-47](https://user-images.githubusercontent.com/73098407/134282227-3fe5030e-e244-4006-bc42-9874111654a8.png)